src/components/MovieTile/Tests/MovieTile.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen, fireEvent } from "@testing-library/react";
+import MovieTile from "../Component/MovieTile";
+
+describe("MovieTile", () => {
+  const movie = {
+    poster_path: "https://example.com/image.jpg",
+    title: "Example Movie",
+    release_date: "2022",
+    genres: ["Action", "Adventure"],
+  };
+
+  const onClick = jest.fn();
+  const onEdit = jest.fn();
+  const onDelete = jest.fn();
+
```

Comment:
This can be extracted into the hook:

  beforeEach(() => {
    render(
      <MovieTile
        movie={movie}
        onClick={onClick}
        onEdit={onEdit}
        onDelete={onDelete}
      />
    );
  });